dropshot/src/handler.rs

```diff
@@ -1285,6 +1316,170 @@ impl From<HttpResponseUpdatedNoContent> for HttpHandlerResult {
     }
 }
 
+#[derive(Serialize, JsonSchema)]
+pub struct NoHeaders {}
+pub struct HttpResponseHeaders<
```

Comment:
This was probably on your list already but it'd be great to have some user doc comments for this.  (Sorry if I missed them elsewhere!)

Reply:
pushed now; sorry that was a local change I made during a self-review that I neglected to push